container.go

```diff
@@ -148,31 +188,13 @@ func renderCheckpoint(ci *containerInfo, containerConfig *metadata.ContainerConf
 
 	row = append(row, containerConfig.OCIRuntime)
 	row = append(row, ci.Created)
-
 	row = append(row, ci.Engine)
-	if ci.IP != "" {
-		header = append(header, "IP")
-		row = append(row, ci.IP)
-	}
-	if ci.MAC != "" {
-		header = append(header, "MAC")
-		row = append(row, ci.MAC)
-	}
-
-	header = append(header, "CHKPT Size")
+	row = append(row, ci.IP)
```

Comment:
Why are you display the MAC address and IP address unconditionally even if it is not set?

Reply:
Right, that makes sense. Tables are just dumb :wink: or it least not helpful here.

I guess for listing multiple checkpoints your approach is correct, but for the single checkpoint view you should make it conditional.

Reply:
Ok, I will try something :)

Reply:
@adrianreber, I made changes to maintain the previous output format when displaying a single checkpoint. What do you think of this approach?